src/apps/core/app.js

```diff
@@ -76,7 +76,8 @@ define(function(require) {
 				mainTemplate = $(self.getTemplate({
 					name: 'app',
 					data: dataTemplate
-				}));
+				})),
+				crossSiteMessaging = monster.config.crossSiteMessaging;
```

Comment:
Please add the corresponding documentation for `crossSiteMessaging` in the `configuration.md` file.

Reply:
Please let me know if the documentation is sufficient or if we should also include an example in there.